models/errors_blackbox_test.go

```diff
@@ -0,0 +1,26 @@
+package models_test
+
+import (
+	"testing"
+
+	"github.com/almighty/almighty-core/models"
+	"github.com/almighty/almighty-core/resource"
+)
+
+func TestNewSimpleError(t *testing.T) {
+	t.Parallel()
+	resource.Require(t, resource.UnitTest)
+	err := models.NewSimpleError("Error reading database values")
+
+	// not sure what assertion to do here
```

Comment:
@sbose78 Sounds boring but you could construct a `SimpleError` object by hand and compare it to the one generate with `NewSimpleError()` using [`assert.EqualValues(t, expectedObj, err)`](https://godoc.org/github.com/stretchr/testify/assert#EqualValues)